components/tidb_query/src/codec/mysql/decimal.rs

```diff
@@ -62,6 +62,46 @@ impl<T> Res<T> {
             _ => false,
         }
     }
+
+    /// Convert `Res` into `Result` with an `EvalContext` that handling the errors
+    /// If `truncated_err` is None, `ctx` will try to handle the default truncated error: `Error::truncated()`,
+    /// otherwise handle the specified error inside `truncated_err`.
+    /// Same does `overflow_err` means.
+    fn into_result_impl(
+        self,
+        ctx: &mut EvalContext,
+        truncated_err: Option<Error>,
```

Comment:
Since `truncated_err` is always `None` in  `into_result_with_overflow_err` and `into_result_with_overflow_err`, do you think we can remove it here?